root/programs/C#/Frameworks/Tools/DaoGen_Tool/Form2.ja-JP.resx

```diff
@@ -215,28 +218,96 @@
+  <data name="cbxTypedDataSet.Size" type="System.Drawing.Size, System.Drawing">
+    <value>112, 17</value>
+  </data>
+  <data name="cbxTypedDataSet.Text" xml:space="preserve">
+    <value>型付きデータセット</value>
+  </data>
+  <data name="gbxDTO.Text" xml:space="preserve">
+    <value>DTOを生成する（エンコーディングはDaoと同じになる</value>
+  </data>
+  <data name="lblEscapeChar.Size" type="System.Drawing.Size, System.Drawing">
+    <value>83, 13</value>
   </data>
   <data name="lblEscapeChar.Text" xml:space="preserve">
     <value>エスケープ文字：</value>
   </data>
-  <data name="lblFamilyName.Text" xml:space="preserve">
-    <value>姓：</value>
+  <data name="chkEscapeToNChar.Size" type="System.Drawing.Size, System.Drawing">
+    <value>164, 17</value>
   </data>
-  <data name="lblSetDaoDefinition.Text" xml:space="preserve">
-    <value>D層定義情報ファイル</value>
+  <data name="chkEscapeToNChar.Text" xml:space="preserve">
+    <value>エスケープ文字をTO_NCHAR</value>
   </data>
-  <data name="lblSetOutput.Text" xml:space="preserve">
-    <value>出力ファイル</value>
+  <data name="label5.Size" type="System.Drawing.Size, System.Drawing">
+    <value>60, 13</value>
   </data>
-  <data name="lblSetSourceTemplate.Text" xml:space="preserve">
-    <value>ソース テンプレート ファイル</value>
+  <data name="label5.Text" xml:space="preserve">
+    <value>LIKE記号：</value>
+  </data>
+  <data name="groupBox1.Text" xml:space="preserve">
+    <value>NCHAR、NVARCHAR2、NCLOB生成オプション</value>
+  </data>
+  <data name="gbxOracleLikeSetting.Text" xml:space="preserve">
+    <value>Like検索設定（ODP.NET）</value>
+  </data>
+  <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
+  <data name="cbxOnlyTableMaintenance.ImeMode" type="System.Windows.Forms.ImeMode, System.Windows.Forms">
+    <value>NoControl</value>
+  </data>
+  <data name="cbxOnlyTableMaintenance.Location" type="System.Drawing.Point, System.Drawing">
+    <value>5, 19</value>
+  </data>
+  <data name="cbxOnlyTableMaintenance.Size" type="System.Drawing.Size, System.Drawing">
+    <value>383, 17</value>
+  </data>
+  <data name="cbxOnlyTableMaintenance.Text" xml:space="preserve">
+    <value>テーブルメンテナンス画面のみ生成します。（Dao, SQL, DTOは生成しません）</value>
+  </data>
+  <data name="groupBox2.Text" xml:space="preserve">
+    <value>その他の設定</value>
+  </data>
+  <data name="tabPage1.Size" type="System.Drawing.Size, System.Drawing">
+    <value>533, 453</value>
   </data>
   <data name="tabPage1.Text" xml:space="preserve">
     <value>ステップ１</value>
   </data>
+  <assembly alias="mscorlib" name="mscorlib, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
+  <data name="cbxTableMaintenance.AutoSize" type="System.Boolean, mscorlib">
+    <value>False</value>
+  </data>
+  <data name="cbxTableMaintenance.Location" type="System.Drawing.Point, System.Drawing">
+    <value>5, 42</value>
+  </data>
+  <data name="cbxTableMaintenance.Text" xml:space="preserve">
+    <value>テーブルメンテナンス画面を生成します。</value>
```

Comment:
@daisukenishino 

>"テーブルメンテナンス画面を生成します。" have a possibility to generate the "DTO and Dao and SQL" depending on the setting. This can be understood by reference to the code.

Which file is the "reference to the code"?